<?xml version="1.0"?>
<sdf version="1.5">
    <model name="stairs">
      <pose> 1.0336  0 &-0.1136/2+H& 0 0 0 </pose> 
      <static>true</static>

      <link name="step1">
        <collision name="collision">
          <geometry>
            <box> <size>0.8672 1.2 0.1136</size> </box>
          </geometry>
        </collision>
        <visual name="visual">
          <geometry>
            <box> <size>0.8672 1.2 0.1136</size> </box>
          </geometry>
        </visual>
      </link>

      <link name="step_edge">
        <pose> -0.4336 0.0 0.0 1.57 0 1.57 </pose>
        <collision name="collision">
          <geometry>
            <box> <size>1.2 0.1136 0.001</size> </box>
          </geometry>
        </collision>
        <visual name="visual">
          <geometry>
            <box> <size>1.2 0.1136 0.001</size> </box>
          </geometry>
        </visual>
      </link>


      <link name="step2">
        <pose>  &L& 0 &H& 0 0 0 </pose>
        <collision name="collision">
          <geometry>
            <box> <size>0.8672 1.2 0.1136</size> </box>
          </geometry>
        </collision>
        <visual name="visual">
          <geometry>
            <box> <size>0.8672 1.2 0.1136</size> </box>
          </geometry>
        </visual>
      </link>

      <link name="step3">
        <pose> &2*L& 0 &2*H& 0 0 0 </pose>
        <collision name="collision">
          <geometry>
            <box> <size>0.8672 1.2 0.1136</size> </box>
          </geometry>
        </collision>
        <visual name="visual">
          <geometry>
            <box> <size>0.8672 1.2 0.1136</size> </box>
          </geometry>
        </visual>
      </link>

      <link name="step4">
        <pose> &3*L& 0 &3*H& 0 0 0 </pose>
        <collision name="collision">
          <geometry>
            <box> <size>0.8672 1.2 0.1136</size> </box>
          </geometry>
        </collision>
        <visual name="visual">
          <geometry>
            <box> <size>0.8672 1.2 0.1136</size> </box>
          </geometry>
        </visual>
      </link>

      <link name="step5">
        <pose> &4*L& 0 &4*H& 0 0 0 </pose>
        <collision name="collision">
          <geometry>
            <box> <size>0.8672 1.2 0.1136</size> </box>
          </geometry>
        </collision>
        <visual name="visual">
          <geometry>
            <box> <size>0.8672 1.2 0.1136</size> </box>
          </geometry>
        </visual>
      </link>

    </model>
</sdf>

